addons/event/models/event_event.py

```diff
@@ -379,6 +380,9 @@ def _compute_from_event_type(self):
                         })
                     for line in event.event_type_id.event_type_ticket_ids]
 
+            if event.event_type_id.tag_ids:
+                event.tag_ids = event.event_type_id.tag_ids
```

Comment:
I wasn't sure about that one either. I guess it could be mildly infuriating to lose all your tags upon changing event type. I'll ask LNA what he prefers

Reply:
Forgot to answer: LNA said he wanted this behaviour

Reply:
Just to keep track: rewrite is ok.